middleware/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var config = require('../config');
+
+module.exports = (req, res, next) => {
+  let accessToken = req.headers['x-access-token'];
```

Comment:
Don't we use 4 spaces for indentation everywhere in the project? (We can change that maybe)